site/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "forceConsistentCasingInFileNames": true,
     "incremental": true,
     "isolatedModules": true,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
As above RE typescript/JSX.

Reply:
Replied in https://github.com/blockprotocol/blockprotocol/pull/441#discussion_r925489980